.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,53 @@
+name: PHPCS
+
+on:
+  pull_request:
+
+jobs:
+  coding-standards:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+
+    name: Coding Standards
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Set up Composer caching
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-composer-dependencies
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, gd
+          tools: composer:v2
+          coverage: none
+
+      - name: Install composer dependencies
+        uses: nick-invision/retry@v1
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          command: composer install
+
+      - name: Run PHPCS
```

Comment:
Recommend validating the composer package with:

```

- name: Validate Composer
        run: composer validate --strict
```

Reply:
This happens in `phpunit.yml`, although I suppose it makes more sense here so it runs once instead of on each of the PHPUnit variations.